source/_templates/layout.html

```diff
@@ -2,6 +2,10 @@
 
 {% block extrahead %}
 {{ super() }}
+<meta name="docsearch:language" content="en"/>
+<meta name="docsearch:version" content="1.8.0"/>
+<meta name="docsearch:branch" content="latest"/>
```

Comment:
These hardcode 1.8.0 and latest. Instead they should use the sphinx variables:

https://github.com/OSC/ood-documentation/blob/9d24e708aa435e8e8a908c474d005c65c9d53e09/source/conf.py#L69-L72

https://github.com/OSC/ood-documentation/blob/9d24e708aa435e8e8a908c474d005c65c9d53e09/source/conf.py#L152

Reply:
You can search in the docs where these variables are used elsewhere.

Reply:
If we only want the meta tags to be in latest then we should have an if block around these meta tags testing if the github_version is latest.